packages/typespec-python/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 | `azure-mgmt-core` dep of generated code (If generating mgmt plane code) | `1.3.2`       |
 | `typing-extensions` dep of generated code (If generating with constants)| `4.0.1`       |
 
+**New Features**
+
+- Add support for `--flavor` flag. Only valid value right now is the `"azure"` flag. When `--flavor=azure` is passed in, we generate an SDK following Microsoft Azure guidelines #2440
```

Comment:
I think "valid value" is slightly misleading, and gives the impression that if you pass something else it will fail. Maybe something like "only special value"?